Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -1486,8 +1486,10 @@ public void openUrlInApp(String url) {
                 Intent httpIntent = new Intent(Intent.ACTION_VIEW);
                 httpIntent.setData(Uri.parse(url));
                 activity.startActivity(httpIntent);
-            } catch (Exception e) {
-                Log.e(LOG_TAG, "Exception occurred while calling WebView", e);
+            } catch (ActivityNotFoundException e) {
+                Toast.makeText(context, "No Activity found to handle Intent", Toast.LENGTH_SHORT).show();
```

Comment:
Can we have a fallback here? No Activity Found we should not show to the users via Toast

Reply:
@witcher-shailesh  What should be the fallback in that case ?

Reply:
Should we open google play store?